packages/syft/setup.cfg

```diff
@@ -120,14 +120,14 @@ test_plugins =
     pytest-asyncio
     pytest-randomly
     pytest-sugar
-    pytest_mock_resources
     python_on_whales
```

Comment:
MAYBE: I think we could also remove unsed dependencies here like
1. python_on_whales
2. lxml
3. joblib (not fully sure if joblib is being used?)

Reply:
and I think pytest-asyncio is also not used

Reply:
agree

Reply:
Lets address in a smaller follow up PR.